tests/unit/Contracts/token_test.py

```diff
@@ -0,0 +1,31 @@
+"""Test uniswap contracts."""
+import pytest
+
+from Arbie import BigNumber
+from Arbie.Contracts import ContractFactory
+from Arbie.Contracts.tokens import GenericToken
+
+
+@pytest.fixture
+def token_factory(w3) -> ContractFactory:
+    return ContractFactory(w3, GenericToken)
+
+
+@pytest.fixture
+def dai(deploy_address, token_factory) -> GenericToken:  # noqa: WPS442
```

Comment:
Create an issue regaridng why these lines need shadowing while uniswap_test dosn't need it.

Reply:
https://github.com/owodunni/arbie/issues/27